pkg/piecestore/psserver/psdb/testdata/sqlite.v1.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE `ttl` (`id` BLOB UNIQUE, `created` INT(10), `expires` INT(10), `size` INT(10));
+CREATE TABLE `bandwidth_agreements` (`satellite` BLOB, `agreement` BLOB, `signature` BLOB, `uplink` BLOB, `serialnum`	TEXT NOT NULL, `total` INT(10), `maxsize` INT(10), `createdunixsec`	INT(10), `expirationunixsec` INT(10), `action` TEXT, `daystartdateunixsec` INT(10));
+CREATE INDEX idx_ttl_expires ON ttl (expires);
+
+-- NEW DATA --
```

Comment:
`NEW DATA` is for inserting new data into tested db. Not needed in this case because we are not adding new table. In this file you need to describe DB state after migration.

Reply:
done